MetaMask/metamask-extension
# feat: New onboarding text

## **Description**

Updates the "Help us improve MetaMask" copy during onboarding.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24050?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2336

## **Manual testing steps**

1. Start onboarding
2. Check that the copy in the 'Help us improve MetaMask' screen matches the new copy.
3. Test the Privacy Policy link

## **Screenshots/Recordings**



### **Before**

<img width="616" alt="Screenshot 2024-04-16 at 12 25 09" src="https://github.com/MetaMask/metamask-extension/assets/11148144/07f310c2-5e41-4245-b00b-2176d4023e22">


### **After**

<img width="835" alt="Screenshot 2024-05-05 at 01 52 00" src="https://github.com/MetaMask/metamask-extension/assets/11148144/098907d5-0176-46ef-93b2-4c11be0d85cf">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
